watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -1407,8 +1407,8 @@ def _flow_mol_phase_comp(self):
 
             def rule_flow_mol_phase_comp(b, p, j):
                 return (
-                    b.flow_mass_phase_comp[p, j]
-                    == b.flow_mol_phase_comp[p, j] * b.params.mw_comp[j]
+                    b.flow_mass_phase_comp[p, j] / b.params.mw_comp[j]
+                    == b.flow_mol_phase_comp[p, j]
```

Comment:
This is changed due to how the WaterTap scaling utilities work. Because the constraint is named `eq_flow_mol_phase_comp`, it is automatically scaled using the scaling factor for `flow_mol_phase_comp`. However, as originally written, it ought to be scaled instead like `flow_mass_phase_comp`. This slight modification of the constraint is easier than messing with that workflow.